README.md

```
@@ -3,498 +3,296 @@
 [`docs/playback.md`](docs/playback.md).
 
-## Try discovery
+---
 
-Balun currently requires Rust 1.98 or newer. Ordinary local discovery is the
-default:
+## Architecture
 
-```bash
-cargo run --locked --bin balun-discover
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)